packages/dita-example-sx-modules-xsd-hazard-domain/src/configureSxModule.js

```diff
@@ -113,7 +114,7 @@ export default function configureSxModule(sxModule) {
 
 	Object.keys(HAZARD_VISUALIZATION_BY_TYPE).forEach(function(hazardType) {
 		var hazardVisualization = HAZARD_VISUALIZATION_BY_TYPE[hazardType];
-		configureProperties(sxModule, 'self::hazardstatement[@type="' + hazardType + '"]', {
+		configureProperties(sxModule, xq`self::hazardstatement[@type="${hazardType}"]`, {
```

Comment:
Those quotes should not be needed: the string interpolation should kick in to make the hazardType a variable